### ISSUE TITLE
die cut bleeds not lining up

### ISSUE THREAD

**Issue by th3tick** [18xx-maker/18xx-maker#216]
<img width="223" alt="Screen Shot 2020-03-02 at 9 33 55 PM" src="https://user-images.githubusercontent.com/48138042/75743279-aa186d00-5ccd-11ea-8f3e-9ca400bc53f1.png">

If G and D tile were swapped, all would be well.

Not sure if this is beyond the capability of the bleed alignment design.

**Comment by kelsin** [comment]
Yeah, def good to have this issue open. The current code only attempts to rotate tiles in order to have track or non-track line up. It can't handle cases like this where there is no rotation of the D tile that works. Next step is reordering track first before printing, which is something we can look at later. For now if you want to solve it ONLY for your game you can put `"group": "individual"` on the definition of the D2 tile. This will make it print on it's own in it's own group with full bleed.
